runit-python/Dockerfile

```diff
@@ -4,6 +4,12 @@ MAINTAINER Socrata <sysadmin@socrata.com>
 RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
     DEBIAN_FRONTEND=noninteractive apt-get -y install python python-dev python-pip
 
+# Update the Python installed version to latest
```

Comment:
I'm concerned that upgrading this version could hurt builds upstream or do something negative.